build.sbt

```diff
@@ -45,6 +45,7 @@ ThisBuild / libraryDependencies ++= Seq(
   mockito % Test)
 
 Test / javaOptions ++= Seq("-Xmx10G", "-XX:+UseG1GC")
+Test / testOptions += Tests.Argument("-oD")
```

Comment:
What is this?

Reply:
This shows the total time spent on each individual test. Since tests take a long time to execute, I was curious to know which tests take more time and, potentially in the future, investigate if we can improve them.

Reply:
Top 👍